react-native/react/native/remote-manager.desktop.js

```diff
@@ -52,12 +50,10 @@ class RemoteManager extends Component {
   }
 
   componentWillMount () {
-    engine.listenOnConnect(() => {
```

Comment:
I probably added this, but it's a mistake. This component shouldn't have to deal with re registering things on connect. That should be handled in actions that know what they want to do onConnect.